### ISSUE TITLE
 File is not included in module tree for all rust files. 

### ISSUE THREAD

**Issue by JoeButler99** [intellij-rust/intellij-rust#6103]
## Environment

* **IntelliJ Rust plugin version:** 0.3.130.3337-202
* **Rust toolchain version:** 1.46.0 (04488afe3 2020-08-24) x86_64-unknown-linux-gnu
* **IDE name and version:** CLion 2020.2.1 (CL-202.6948.80)
* **Operating system:** Linux 5.7.9-1-manjaro

## Problem description

All rust files show the "File is not included error". I've tried all settings under Settings -> Languages -> Rust & Cargo.
Have reattached Cargo. Invlidated Caches, deleted .idea folder and many combinations of above.

I even started a new binary project and the main.rs also shows this error.
I also tried in Intellij, it brings the same error. 

All cargo commands working from CLI
I tested with VS Code on a working project. Had no issues. 
I upgraded Clion from 2020.1.1 to 2020.2.1, no change. 

Basically I working setup from Cargo and other Editors, but both CLion and Intellij absolutely refuse to pick up and analyse the files.

## Steps to reproduce

In my case just creating a new empty rust project will throw the File is not included in module tree error in the main.rs

**Comment by aknarts** [comment]
I have the same behavior in lib.rs and other modules.

One note, old project was behaving ok till I cleaned out .idea directory in it, than the same manifested

**Comment by aknarts** [comment]
Seems to be fixed in 0.3.131.3366

**Comment by vlad20012** [comment]
Oh, I'm sorry, I missed the issue. Most likely it duplicates #5962 (the fix is just released, yes). Do you have a workspace under symlink?

**Comment by aknarts** [comment]
Mine was under symlink, cannot speak for the original reporter.

**Comment by JoeButler99** [comment]
yes thankyou - mine is fixed with update https://intellij-rust.github.io/2020/09/21/changelog-131.html

I do have a symlink a bit further up the path, this release has fixed.

**Comment by Undin** [comment]
Fixed in #6075

**Comment by Bjohnson131** [comment]
Still seeing in 4.178.4873-222

**Comment by Undin** [comment]
@Bjohnson131 could you create a new issue with mode details (reproducible example, for instance)? Comments in such old issues without details usually are not very useful because most likely the current problems with the similar symptoms have other causes
